spec/api-app-spec.js

```diff
@@ -357,4 +357,66 @@ describe('app module', function () {
       assert.equal(app.getPath('music'), __dirname)
     })
   })
+
+  describe('getFileIcon() API', function () {
+    const iconPath = path.join(process.cwd(), 'fixtures/assets/icon.ico')
+    const sizes = {
+      small: 16,
+      normal: 32,
+      large: 48
+    }
+
+    it('fetches non-empty icon', function (done) {
```

Comment:
The other specs are `fetches small icons` and fetches normal icons` though, so either this should be `fetches non-empty icons` or `fetches a non-empty icon`. Currently the grammar is a little off.